test/tzdata/version.jl

```diff
@@ -51,16 +63,22 @@ mktempdir() do temp_dir
     @test_throws ErrorException tzdata_version_dir(dirname(@__FILE__))
 end
 
-@test tzdata_version_archive(archive) == TZDATA_VERSION
-@test_throws ProcessFailedException tzdata_version_archive(@__FILE__) == TZDATA_VERSION
+use_artifacts = VERSION >= v"1.4"
```

Comment:
Would make more sense to put this earlier in the file

Reply:
Sure.